static/src/stylesheets/module/content-garnett/_article-immersive.scss

```diff
@@ -567,3 +560,191 @@
         }
     }
 }
+
+//overides the immersive article — done like this to keep gallery's and photo eassy in ok shape
+.content--immersive:not(.immersive-main-media__gallery) {
+    &.immersive-main-media {
+        @include mq(wide) {
+            min-height: 600px;
+        }
+    }
+    .content__article-body {
+        @include mq($from: desktop) {
+            margin-left: -$gs-gutter / 2;
+        }
+    }
+    .immersive-main-media__headline-container {
+        margin-top: -($gs-column-width * 2) + 4px;
```

Comment:
` -($gs-column-width * 2) + 4px` ? lets just say `-124px`. I don't like the magic numbers, but at least lets be clear about it 😄